### ISSUE TITLE
REST API like SES

### ISSUE THREAD

**Issue by ishanjain28** [freeCodeCamp/mail-for-good#209]
It's a great tool if I want to write a message and send it to a lot of people but having a REST API where I can send data like `to`, `from`, `subject`, `body` like the AWS SES API and it sends that message to the specified email would be great.

**Comment by ishanjain28** [comment]
@karuppiah7890 I checked the code and in [secrets.js](https://github.com/freeCodeCamp/mail-for-good/blob/master/server/config/secrets.js) it reads smtpServer configuration from env variables but doesn't uses them anywhere. 

And in [planned_features.md](https://github.com/freeCodeCamp/mail-for-good/blob/master/docs/planned_features.md), They have mentioned they'll be building a REST API for it so you can use it to do exactly what I said.

**Comment by ishanjain28** [comment]
TL;DR, This tool apparently will also enable you to run a SMTP Server so you don't need AWS or any other third party provider to send emails.

**Comment by karuppiah7890** [comment]
@ishanjain28 I believe that is for some testing, where they use a SES simulator. And the simulator is only for testing.

And also, about the REST APIs, yes, I did think you might be referring to that feature, but that feature is about providing an API to send campaigns to lists unlike sending emails to a specific `to` address. And I believe issue #109 is about this. So, I think we can close this issue

**Comment by ishanjain28** [comment]
Thanks for clearing it up. I understand it a bit better now. :)

**Comment by karuppiah7890** [comment]
@ishanjain28 And no, the tool doesn't enable you to run a SMTP server. M4G is only an email campaign management tool, it's not a mail server, and that's why we use AWS SES and only that is supported now nothing else and most probably, it will remain the same in the  future too. You can check issue #161 to know more about this. And some people have been asking if they could use external SMTP servers, check issue #205 , #208

**Comment by manjeshpv** [comment]
open source email server compatible with Amazon SES APIs https://github.com/manjeshpv/emailq works with SMTP
